vllm/v1/attention/backends/cpu_attn.py

```diff
@@ -591,6 +591,10 @@ def forward(
                                        attn_type=attn_type)
             else:
                 # prefix-enabled attention
+                if not _use_ipex:
+                    raise NotImplementedError(
+                        "Chunked prefill on CPU requires "
+                        "intel_extension_for_pytorch.")
```

Comment:
Why is this not necessary for the other CPU architectures which don't support chunked prefill?